src/components/course/data/utils.jsx

```diff
@@ -357,13 +358,17 @@ export function findHighestLevelSku({ courseEntitlements, seats }) {
   return findHighestLevelSeatSku(seats) || findHighestLevelEntitlementSku(courseEntitlements);
 }
 
+export function isActiveSubscriptionLicense(subscriptionLicense) {
```

Comment:
nit: Should there be a unit test (or other means of getting coverage) on this function?

Reply:
Yep.  I kind of assumed it would get covered as part of other tests, but apparently it didn't.